packages/mutation-testing-metrics-scala/.npmignore

```diff
@@ -0,0 +1,6 @@
+**/*
```

Comment:
Why does this need a npmignore?

Reply:
It's still being published on npm unfortunately. I haven't found a way to fix it while still having lerna start the sbt publishing process. So at least now it doesn't publish all the jars to npm